assets/app/styles/_create.less

```diff
@@ -0,0 +1,225 @@
+//
+// Create App
```

Comment:
I'm guilty of the same thing, need to break up my PRs :grimacing:
On the up side, I'm excited to see huge files broken down, will make rebases / maintenance much more manageable.